aries-test-harness/features/0183-revocation.feature

```diff
@@ -1,4 +1,4 @@
-@revocation @AIP20
```

Comment:
Can you add the `@AIP20` back in to the feature file? Though revocation isn't technically part of any AIP level, we decided that AIP20 made the most sense for ease of execution so it would be included with run sets that only say run AIP20 tests. Thanks.

Reply:
This is quite unfortunate for aries-vcx as it generally doesn't support AIP20 yet but it does support revocation, so it would be desirable run revocation tests by `-t @revocation` and  exclude AIP20 tests by `-t ~@AIP20`.

In order to revert this change and still run revocation tests, I will have to not use `-t ~@AIP20` and hence will have to manually exclude all the matched AIP20 tests (and update when new matched AIP20 test is added), or to specify the tests I want to run one by one.

Both of those are doable, but quite impractical. On top of that, as you say, revocation is not part of any AIP level in the first place.

That being said, I realize this change would affect other pipelines and so I will work around it.